### PR TITLE
AGENT-325: Set AgentClusterInstall networkType using IP_STACK

### PR DESCRIPTION
**Pull request by rwsu**
networkType is set to OpenShiftSDN if IP_STACK == v4

networkType is set to OVNKubernetes if IP_STACK == v4v6|v6

### CODE REVIEW

File: agent/04_agent_configure.sh [hunk 1]
```diff
@@ -110,6 +110,20 @@ cat >> "${MANIFESTS_PATH}/agent-cluster-install.yaml" << EOF
...
+    networkType: OVNKubernetes
+EOF
+    ;;
+  *)
+cat >> "${MANIFESTS_PATH}/agent-cluster-install.yaml" << EOF
+    networkType: OpenShiftSDN
+EOF
+    ;;
+esac
+cat >> "${MANIFESTS_PATH}/agent-cluster-install.yaml" << EOF
```
Comment (andfasano) [inline agent/04_agent_configure.sh:126]: @rwsu what about setting the field just as in https://github.com/openshift-metal3/dev-scripts/blob/522d3c853a36c44125ea165cf8e633c5e88d7693/ocp_install_env.sh#L218? `NETWORK_TYPE` should already contain the right value based on IP_STACK

Comment (rwsu) [reply agent/04_agent_configure.sh:126]: Ah. Even better. I'll make that change. Thanks!

**Pull request merged**